packages/expo-cli/src/commands/build/BaseBuilder.ts

```diff
@@ -124,10 +125,16 @@ export default class BaseBuilder {
       return;
     }
 
+    const { canPurchasePriorityBuilds, numberOfRemainingPriorityBuilds } = buildStatus;
+    invariant(
+      canPurchasePriorityBuilds !== undefined && numberOfRemainingPriorityBuilds !== undefined,
+      'canPurchasePriorityBuilds and numberOfRemainingPriorityBuilds should be defined when there are build jobs'
+    );
```

Comment:
I'm wary about this invariant since these fields are basically deprecated and I imagine we'll remove them.